CMakeLists.txt

```diff
@@ -23,16 +23,27 @@
 #
 #=============================================================================
 
-cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5.2 FATAL_ERROR)
```

Comment:
This is still to fix (at least). Why bump the version up?

Reply:
<details >
<summary>Versions not less than this one are widely available according to repology data:</summary>

[![](https://repology.org/badge/vertical-allrepos/cmake.svg?minversion=3.5.0)](https://repology.org/project/cmake/versions)

</details>

Reply:
d94c3a5e647e64876c692a470ba62e5b703735d0